app/models/course/lesson_plan_item.rb

```diff
@@ -7,6 +7,8 @@ class Course::LessonPlanItem < ActiveRecord::Base
   validates :base_exp, numericality: { only_integer: true }
   validates :time_bonus_exp, numericality: { only_integer: true }
   validates :extra_bonus_exp, numericality: { only_integer: true }
+  validates :start_time, presence: true
```

Comment:
You don't actually need these validations if your database schema already says that the column is not nullable.